### ISSUE TITLE
saveFrames() files don't appear in left nav

### ISSUE THREAD

**Issue by mimiyin** [processing/p5.js-web-editor#86]
Perhaps this is really a feature request. Are the files stored but simply not showing up in the left nav?

**Comment by catarak** [comment]
I think this is a feature request. From a technical perspective, I think it's unclear what should happen. If the frames are to appear in the left sidebar, then each one would need to be uploaded to S3, which to me doesn't appear to be the right approach because $$$. To me it would make sense if the frames are downloaded to a user's computer, since that's the place they would be using them anyway. 

Thoughts?
@shiffman @lmccart

**Comment by mimiyin** [comment]
Yes definitely $$$. What about arbitrary cap? Speaking of which, is there a cap for uploading data? People sometimes want to re-use the saved images immediately in the same sketch. Supporting a modest amount of prototyping ability or teaching-example ability would be useful.

**Comment by lmccart** [comment]
this functionality isn't really supported in the library (I think this function should actually be removed completely because it's sort of misleading... save() pretty much does the trick), so we might consider skipping it in the editor as well if it creates complications.

**Comment by shiffman** [comment]
We can think about this in the future, but right now I don't think we should add functionality that only works in the web editor.   Someday having an add-on library perhaps combined with node or even an editor-plug-in for rendering of sketches to images / movies could be a great project for summer of code, etc.   In this case the web editor should do whatever the browser / p5.js would normally do which is just send the file to the downloads folder of the client machine.
